pylibs/stress_tests/README.md

```diff
@@ -0,0 +1,226 @@
+# Stress Tests
+
+OpenThread Network Simulator (OTNS) is used to run stress tests to improve the robustness of OpenThread.
+
+## Network Forming
+
+### Variable Sized Network Forming
```

Comment:
While this file can work as is, I think it would be much more readable in the Markdown viewer if each test could be summarized in a single list or table.  With everything as headers the text gets a little lost.

For example, this is bit more readable:

### Variable Sized Network Forming

Test Purpose — Make sure different number of nodes can form a single partition within reasonable time limits.

- Topology
  - NxN Routers, for n in 1 to 7.
- Procedure
  1. Boot up NxN Routers.
  1. Wait for them to form a single partition.
- Fault Injections
  - None
- Pass Criteria — Network formation time is less than corresponding time limits.